pkg/datastore/datastore.go

```diff
@@ -16,6 +16,12 @@ type DataStore interface {
 
 	// Find executes a find command and returns an iterator over the matching items.
 	Find(ctx context.Context, kind string) (Iterator, error)
+
+	FindOne(ctx context.Context, kind, name string) (Iterator, error)
+
+	IsExist(ctx context.Context, kind, name string) (bool, error)
+
+	Update(ctx context.Context, kind, name string, entity interface{}) error
```

Comment:
In https://github.com/oam-dev/velacp/pull/94 it introduces Add() method and Put() is now equivalent to Update().

Please remove Update() and just use Put()

Reply:
done